airbyte-scheduler/src/main/resources/tables/Jobs.yaml

```diff
@@ -0,0 +1,36 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+title: Jobs
+description: representation of a jobs record.
+type: object
+required:
+  - id
+  - scope
+  - config
+  - status
+  - created_at
+  - updated_at
+additionalProperties: false
+properties:
+  # First column defined is the primary Key
+  id:
+    type: number
```

Comment:
So if I am not wrong, these lines defining the types are used by the JsonValidator to ensure Json schema validity right?

Is it possible/desirable if we could we add additional columns `sqltype` such as these:
```
properties:
  # First column defined is the primary Key
  id:
    type: number
    sqltype: BIGSERIAL
  scope:
    type: string
    sqltype: VARCHAR(255)
  config_type:
    type: string
    sqltype: JOB_CONFIG_TYPE
```

Then when I import data back to the Database, I can actually reuse whatever textValue is written in the schema files in the sqltype field instead of trying to convert from JsonNodeType to SQL type?

And then we could more easily reproduce perfectly the initial init script:
`airbyte-db/src/main/resources/schema.sql`

WDYT?

Reply:
If we get rid of the schema swap can we be done with this project? Like it seems like otherwise we are able to pull data out of the json schema and put it in the existing tables. I really want us to close the book on this and avoid it creeping anymore.

Reply:
Sorry, I haven't completely removed the schema swap yet but it is very straightforward to do so.

I am just thinking we could benefit from keeping that mechanism so we don't rely on a complete wipe through `docker-compose down -v`.

With this PR, we are now able to recreate a clean schema using part of the `schema.sql` if that's acceptable and re-import the old data into the tables. Therefore we don't lose the workspaces logs etc (which was nice to have points)